### ISSUE TITLE
Import is undefined with default create-react-app setup

### ISSUE THREAD

**Issue by TheAncientGoat** [wasmerio/wasmer-js#155]
Using a `create-react-app` bootstrapped application, when trying to import the module as shown in the docs, the `WASI` object is undefined. 

```javascript
import { WASI } from '@wasmer/wasi';

console.log(WASI); // prints "undefined"
```

Full sample code here: (see App.js)
https://repl.it/repls/NeglectedIrritatingNumerators

Not sure if it's a webpack issue, or babel issue, but as create-react-app is a pretty standard setup, it would be good to check that it is supported.

I have also tried with parcel + the create-react-app --typescript ; same results, and for `wasmer-js` versions 0.3.2 - 0.4.5

**Comment by syrusakbary** [comment]
Hi @TheAncientGoat,

We are working on a fix. In order to know if what we have in mind will fix it, can you try the following and comment here if it worked?

```js
import { WASI } from '@wasmer/wasi/lib';

console.log(WASI); // prints "undefined"
```

**Comment by torch2424** [comment]
Fixed in Verion `0.4.6`.

Thank you very much for finding and filing the issue @TheAncientGoat ! 😄

**Comment by TheAncientGoat** [comment]
Tested and it works, thanks for the fast fix everyone :)
